### DISCUSSION TITLE
how to use Filament::pushMeta  in version 3

### DISCUSSION THREAD

**Discussion by omarmm** [filamentphp/filament#8692] [General]
how to use Filament::pushMeta  in version 3 ?
I get an error :
Call to undefined method Filament\FilamentManager::pushMeta()

in version 2 it was working normally, could you please help me?

**Comment by pboivin** [comment]
I think you can use the `panels::head.start` render hook to do the same in v3:
```php
FilamentView::registerRenderHook(
    'panels::head.start',
    fn (): string => '<meta ... >',
);
```
https://filamentphp.com/docs/3.x/support/render-hooks
